File: fix_template.js
const fs = require('fs');
let c = fs.readFileSync('src/components/idc/FieldPool.vue', 'utf8');
const old = ':class="`fp-zone-dot--${getFieldZone(field)}`"';
const rep = ':class="\'fp-zone-dot--\' + getFieldZone(field)"';
if (c.includes(old)) {
  c = c.replace(old, rep);
  fs.writeFileSync('src/components/idc/FieldPool.vue', c);
  console.log('replaced');
} else {
  console.log('not found');
  const lines = c.split('\n');
  lines.forEach((l, i) => { if (l.includes('fp-zone-dot--')) console.log(i+1, l); });
}

File: fix_template.py
content = open(r'd:\liuruxian\project\BrandRadarFront\src\components\idc\FieldPool.vue', 'r', encoding='utf-8').read()
old = ':class="`fp-zone-dot--${getFieldZone(field)}`"'
new = ":class=\"'fp-zone-dot--' + getFieldZone(field)\""
content = content.replace(old, new)
open(r'd:\liuruxian\project\BrandRadarFront\src\components\idc\FieldPool.vue', 'w', encoding='utf-8').write(content)
print('done')
